quickwit/rustfmt.toml

```diff
@@ -6,6 +6,6 @@ comment_width = 120
 format_strings = true
 group_imports = "StdExternalCrate"
 imports_granularity = "Module"
-normalize_comments = false
+normalize_comments = true
```

Comment:
I think this is a bad idea. `/* */` are very useful when commenting out code blocks.

Reply:
I just reverted your previous commit, but I can put it back to `false`.